lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -84,6 +89,7 @@ class CircleImageBase extends NodeBase {
       this.width = width;
       this.height = height;
       this.radius = 0.5 * this.width;
+      this.refreshNeeded = false;
```

Comment:
Ah, you mean: why is not `hover` and `selected` checked for triggering the resize calculation, just like with the rest of the shapes.

The direct answer is: I don't know, this is how the code worked before I added the changes and I did not want to change the logic too much.
I'll look at the code to see if I can understand this myself.